joplin/templates/joplin_UI/primary_navigation.html

```diff
@@ -0,0 +1,31 @@
+{% load staticfiles wagtailadmin_tags %}
+{% load i18n %}
+
+{% block content %}
+
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Correct  me if I'm wrong, but I think this import shouldn't be needed if its already in base.html?

Reply:
You are correct. Good catch. So, what i'm gonna do is remove them from both the base.html and the primary_navigation.html. Then, put them in the skeleton.html in the header.